profiles/sway.py

```diff
@@ -13,13 +13,13 @@ def _prep_function(*args, **kwargs):
 	other code in this stage. So it's a safe way to ask the user
 	for more input before any other installer steps start.
 	"""
-	if "nvidia" in _gfx_driver_packages:
+	__builtins__['_gfx_driver_packages'] = archinstall.select_driver()
```

Comment:
This driver will never be installed unless `xorg` is also installed. Which I can't see that sway does?

Reply:
Wayland uses hardware acceleration as far as I know. Graphics drivers provide that and I believe the same ones are used for Wayland and X